rest/data/patch_test.go

```diff
@@ -144,8 +163,13 @@ func (s *PatchConnectorFetchByProjectSuite) TestFetchTooFew() {
 	s.Equal(s.time.Add(time.Second*10), *patches[0].CreateTime)
 }
 
-func (s *PatchConnectorFetchByProjectSuite) TestFetchNonexistentFail() {
-	patches, err := s.ctx.FindPatchesByProject("zzz", s.time, 1)
+func (s *PatchConnectorFetchByProjectSuite) TestProjectNonexistentFail() {
+	_, err := s.ctx.FindPatchesByProject("zzz", s.time, 1)
+	s.Error(err)
+}
+
+func (s *PatchConnectorFetchByProjectSuite) TestPatchNonexistentFail() {
```

Comment:
"PatchNonexistentFail" seems wrong; we want it to succeed. Maybe "EmptyPatchesOkay" ?

Reply:
done